packages/forms/resources/views/components/text-input.blade.php

```diff
@@ -71,6 +73,7 @@
                 {{ $getExtraInputAttributeBag()->class([
                     'block w-full transition duration-75 rounded-lg shadow-sm focus:border-primary-600 focus:ring-1 focus:ring-inset focus:ring-primary-600 disabled:opacity-70',
                     'dark:bg-gray-700 dark:text-white dark:focus:border-primary-600' => config('forms.dark_mode'),
+                    $getPrefixIcon ? 'pl-10' : '',
```

Comment:
Forgot that it would also make sense to put pr-10 if a suffixIcon is present

Reply:
you should make another class for `rtl` please change `pl-10` to `ltr:pl-10 rtl:pr-10`